content/authentication/next-steps/bonus-module/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "BONUS: User Roles & Spring Security"
+date: 2023-11-14T09:28:27-05:00
+draft: false
+weight: 1
+originalAuthor: Ben Clark # to be set by page creator
+originalAuthorGitHub: brclark # to be set by page creator
+reviewer:  # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+This section contains 5 lessons to build out the **authentication**
+and **authorization** features in Coding Events. The video shows the
+final features added as a result of the lessons. You will add user-owned
+data, refactored controllers to make use of services, refactored
+authentication and authorization using Spring Security framework,
+and roles & privileges to restrict access to the Coding Events based
+on assigned user roles.
+
+### TODO: Add Embedded Intro video
```

Comment:
I think including the preview video that I made might be useful here, to show students what they will be building by the end of the walkthrough